Src/Sarif.PatternMatcher.Cli/StressCommand.cs

```diff
@@ -232,24 +291,44 @@ private List<string> GetWhatFilesToSearch(StressOptions options)
             return filesToSearch;
         }
 
-        private void ExportSizeAndExecutionTime(TimeSpan totalRunTime, string outputFilePath)
+        private void ExportSizeAndExecutionTime(TimeSpan totalRunTime, StressOptions options)
         {
-            // output Tuple list to csv in 3 columns
-            var sb = new StringBuilder($"Filename, File Size in KB, Runtime in ms, Total RunTime: {totalRunTime}{Environment.NewLine}");
+            var perFileSb = new StringBuilder($"Filename, File Size in KB, Runtime in ms, %-CPU Utilization, Peak Mem (MB){Environment.NewLine}");
+            var aggregatedSb = new StringBuilder();
+
+            double meanAnalysisRate = 0;
+            double meanCpuUtil = 0;
 
-            foreach (Tuple<string, long, long> runData in fileDataTupleList)
+            foreach (Tuple<string, long, long, double, double> runData in fileDataTupleList)
```

Comment:
Again here with Tuple instead of just having a class for Performance data.
Seeing runData.Item1, runData.Item2 is not so clear.

There are no comments here other than looking at the StringBuilder line that indicate what they are.

Reply:
@schlaman-ms Should be more explanatory now.